### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by Albinpirela**
# feature/setup branch🌳
## Add readme project
Screenshot 2023-02-11 092424

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```
Comment (hendridg) [inline .github/workflows/linters.yml]: Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
```
Comment (hendridg) [inline .github/workflows/linters.yml]: Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install hint --save-dev
+          [ -f .hintrc ]
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-18.04
```
Comment (hendridg) [inline .github/workflows/linters.yml]: Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

**Review by hendridg — CHANGES_REQUESTED** [review]
# :recycle: Changes required :recycle: 

Hi @Albinpirela , great job!

Please changes some stuff I suggest below

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Albinpirela) [reply .github/workflows/linters.yml:11]: ok thenks

File: .github/workflows/linters.yml (hunk 3 above)
Comment (Albinpirela) [reply .github/workflows/linters.yml]: ok thanks

File: .github/workflows/linters.yml (hunk 2 above)
Comment (Albinpirela) [reply .github/workflows/linters.yml]: ok thanks

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Albinpirela) [reply .github/workflows/linters.yml]: ok thanks

**Review by hendridg — APPROVED** [review]
lgtm!

**Pull request merged**